Bottom: 7f6907fb188abab268d7ca1a889162851cdce1c6
Top:    5a26f8eb49e30d167902e4b5874e48ea3274cbab
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:17:24 +0000

Refresh of and-working-again-mostly

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 2206b3a..a6c7cab 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -56,7 +56,7 @@ presentation_style_assignments_from_appearance (step_file *step, appearance *app
 }
 
 static step_file *
-start_ap214_file (char *filename)
+start_ap214_file (const char *filename)
 {
   FILE *f;
   time_t currenttime;
@@ -341,7 +341,7 @@ object3d_to_step_fragment (step_file *step, object3d *object, char *part_id, cha
 }
 
 void
-object3d_list_export_to_step_assy (GList *objects, char *filename)
+object3d_list_export_to_step_assy (GList *objects, const char *filename)
 {
   step_file *step;
   step_id comp_shape_definition_representation;
@@ -556,7 +556,7 @@ object3d_list_export_to_step_assy (GList *objects, char *filename)
 }
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   step_file *step;
 
diff --git a/src/hid/step/object3d_step.h b/src/hid/step/object3d_step.h
index 740937d..5dc6f4d 100644
--- a/src/hid/step/object3d_step.h
+++ b/src/hid/step/object3d_step.h
@@ -1,2 +1,2 @@
-void object3d_list_export_to_step_assy (GList *objects, char *filename);
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_list_export_to_step_assy (GList *objects, const char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
